cfme/fixtures/v2v.py

```diff
@@ -191,6 +193,35 @@ def _form_data(source_provider, provider):
     return form_data
 
 
+@pytest.fixture(scope="function")
+def form_data_vm_map_obj_mini(request, appliance, source_provider, provider):
+    """Fixture which provides minimal form_data, vm and map object structure for migration plan"""
+    form_data = _form_data(source_provider, provider)
+    source_datastores_list = source_provider.data.get("datastores", [])
+    source_datastore = [d.name for d in source_datastores_list if d.type == "nfs"][0]
+    collection = source_provider.appliance.provider_based_collection(source_provider)
+    vm_name = random_vm_name("v2v-auto")
+    vm_obj = collection.instantiate(
+        vm_name, source_provider, template_name=rhel7_minimal(source_provider)["name"]
+    )
+    vm_obj.create_on_provider(
+        timeout=2400,
+        find_in_cfme=True,
+        allow_skip="default",
+        datastore=source_datastore,
+        power_on=True
+    )
```

Comment:
Source datastore and creating vm can be moved to a fixture .



    def create_vm_on_provider():
         source_datastores_list = source_provider.data.get("datastores", [])
         source_datastore = [d.name for d in source_datastores_list if d.type == "nfs"][0]
        collection = source_provider.appliance.provider_based_collection(source_provider)
        vm_name = random_vm_name("v2v-auto")
        vm_obj = collection.instantiate(
        vm_name, source_provider, template_name=rhel7_minimal(source_provider)["name"])
        vm_obj.create_on_provider(
              timeout=2400,
              find_in_cfme=True,
              allow_skip="default",
              datastore=source_datastore,
              power_on=True
        )
    yield vm
    vm_obj.cleanup_on_provider()
    infrastructure_mapping_collection.delete(mapping)